global-variables.tf

```diff
@@ -1,7 +1,10 @@
+################################################################################
 # Very frequently they deprecate older images, and we need to change this
 # terraform is configured to not recreate images if this variable changes
+################################################################################
+
 variable "image_ubuntu_22" {
-  default = "26459ddd-3ad4-42c0-9255-fc5e682bb037"
+  default = "ad091b52-742f-469e-8f3c-fd81cadf0743"
```

Comment:
Hm. Doesn't this cause terraform to try to destroy all machines in ubuntu 22?

Reply:
@cintiadr Yes that would recreate the machines.

The planned approach was to change the image and try creating a new machine by running terraform commands locally. I'm going to undo this and rethink of another way. I guess it starts with checking if the existing image is deprecated or not as that would determine if changing this is necessary. 

But I'll tread carefully on this. We wouldn't like a situation where all machines are being updated.